package.json

```diff
@@ -88,7 +88,8 @@
   },
   "peerDependencies": {
     "apollo-client": "^2.3.8",
-    "react": "0.14.x || 15.* || ^15.0.0 || ^16.0.0",
```

Comment:
@hwillson @jbaxleyiii Is it reasonable to stop advertising support for `react@0.14.x` at this point? That version was last published two years ago (0.14.9). Also, this is just a peer dependency, not actually a guarantee that `react-apollo` works with `react@0.14.x`, which we probably don't anymore…

Reply:
@benjamn Quite reasonable!